docs/package_templates/autotools_package/all/conanfile.py

```diff
@@ -11,7 +11,7 @@
 import os
 
 
-required_conan_version = ">=1.54.0"
+required_conan_version = ">=2.0.9"
```

Comment:
Just out of curiosity - why 2.0.9, specifically?

Reply:
implements was introduced on that version: https://docs.conan.io/2/changelog.html#jul-2023

Before that, it will not have effect and Conan 2.x will not warn users.